spec.html

```diff
@@ -675,8 +675,31 @@ <h1>Iterator.prototype.find ( _fn_ )</h1>
 
       <emu-clause id="sec-iteratorprototype-@@tostringtag">
         <h1>Iterator.prototype [ @@toStringTag ]</h1>
-        <p>The initial value of the @@toStringTag property is the String value "Iterator".</p>
-        <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.</p>
+        <p>`Iterator.prototype[@@toStringTag]` is an accessor property with attributes { [[Enumerable]]: *false*, [[Configurable]]: *true* }. The [[Get]] and [[Set]] attributes are defined as follows:</p>
+
+        <emu-clause id="sec-get-iteratorprototype-@@tostringtag">
+          <h1>get Iterator.prototype [ @@toStringTag ]</h1>
+          <p>The value of the [[Get]] attribute is a built-in function that requires no arguments. It performs the following steps when called:</p>
+          <emu-alg>
+            1. Return *"Iterator"*.
```

Comment:
seems like a nice opportunity to throw if the receiver isn't a proper iterator?

Reply:
We're only trying to change the set behaviour. We want the get behaviour to look as much like a data property access as possible.